From: Mark Tiefenbruck <mark@fluxbox.org>
Date: Sun, 12 Oct 2008 16:54:35 +0000 (-0700)
Subject: focus window when done dragging to a new workspace with outline moving
X-Git-Url: http://git.fluxbox.org/?p=fluxbox.git;a=commitdiff_plain;h=e5fd401f4eadef1aa4ab91b11d38653d1a4b7194

focus window when done dragging to a new workspace with outline moving
---

diff --git a/src/Window.cc b/src/Window.cc
index e29e761..d23bf44 100644
--- a/src/Window.cc
+++ src/Window.cc
@@ -2861,11 +2861,9 @@ void FluxboxWindow::stopMoving(bool interrupted) {
                                frame().height() + 2*frame().window().borderWidth()-1);
         if (!interrupted) {
             moveResize(m_last_move_x, m_last_move_y, frame().width(), frame().height());
-            if (m_workspace_number != screen().currentWorkspaceID()) {
-                screen().reassociateWindow(this, screen().currentWorkspaceID(), true);
-                frame().show();
-                focus();
-            }
+            if (m_workspace_number != screen().currentWorkspaceID())
+                screen().sendToWorkspace(screen().currentWorkspaceID(), this);
+            focus();
         }
         fluxbox->ungrab();
     } else if (!interrupted) {
